src/worker.py

```
@@ -464,22 +465,146 @@ def _is_basic_auth_valid(req, env) -> bool:
+        f'<p><a href="{link}" style="color:#0d9488;word-break:break-all;">{link}</a></p>'
+        '<p style="font-size:13px;color:#9ca3af;">This link expires in 24&nbsp;hours. '
+        "If you did not create an account, you can safely ignore this email.</p>"
+        '<hr style="border:none;border-top:1px solid #e5e7eb;margin:24px 0;">'
+        '<p style="font-size:12px;color:#9ca3af;text-align:center;">'
+        "Thank you for using alphaonelabs.com!<br>alphaonelabs.com</p>"
+        "</body></html>"
+    )
+    return await _send_email_via_mailgun(to_email, subject, html, env)
+
+
+async def send_password_reset_email(to_email: str, username: str, token: str, env) -> bool:
```

Ruff (0.15.15)

[warning] Unused function argument: `username`

(ARG001)